Matveibos/hometask-4/my_array_rotate.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# the module extends the class Array
+module Addition
+  def lab_rotate(num = -1)
+    arr = clone
```

Comment:
You don't need to assign result of the method to the variable if it is used only once, just provide `clone` as an argument to the `moving` method